README.md

```diff
@@ -177,16 +177,7 @@ TBD
 
 ### Installing and running on macOS via Homebrew
```

Comment:
probably you have to remove this as well?

Or add a TBD after it, just like for the other incomplete sections

Reply:
Good catch.  Corrected.

Reply:
Adam just remove the whole Homebrew section from docs altogether